proto/vtgate.proto

```diff
@@ -549,17 +549,20 @@ message SplitQueryRequest {
   // by the shards are merged and returned to the caller.
   // Two algorithms are supported:
   //  EQUAL_SPLITS
-  //    If this algorithm is used then only one split_column is allowed.
-  //    Additionally, the split_column must have numeric type (integral or
-  //    floating point). The algorithm works by taking the interval [min, max],
-  //    where min and max are the minimum and maximum values of of the
-  //    split_column column in the table-shard, respectively, and
-  //    partitioning it into split_count sub-intervals of equal size.
-  //    The added WHERE clause of each query-part restricts that part to rows
-  //    whose value in split_column belongs to a particular sub-interval.
-  //    This is fast, but requires that the distribution of values of
-  //    split_column be uniform in [min, max], for the number of rows
-  //    returned by each query part to be roughly the same.
+  //    If this algorithm is selected then only the first split_column given is
+  //    used (or the first primary key column if no split_column's are
+  //    specified). In the rest of this algorithm's description, we refer to
+  //    this column as "the split column".
+  //    The split_column must have numeric type (integral or floating point).
+  //    The algorithm works by taking the interval [min, max], where min and
+  //    max are the minimum and maximum values of of the split_column in
+  //    the table-shard, respectively, and partitioning it into split_count
+  //    sub-intervals of equal size. The added WHERE clause of each query-part
+  //    restricts that part to rows whose value in the split_column belongs to
```

Comment:
You are mixing underscore (split_column) and hyphen (sub-intervals, table-shard). You only need one way to group a multi-word term.